orbit/cmd/orbit/orbit.go

```diff
@@ -408,13 +446,32 @@ func main() {
 		// Create an osquery runner with the provided options.
 		r, err := osquery.NewRunner(osquerydPath, options...)
 		if err != nil {
-			return fmt.Errorf("failed to create osquery runner: %w", err)
+			return fmt.Errorf("create osquery runner: %w", err)
 		}
 		g.Add(r.Execute, r.Interrupt)
 
 		ext := table.NewRunner(r.ExtensionSocketPath())
 		g.Add(ext.Execute, ext.Interrupt)
 
+		if runtime.GOOS == "darwin" && c.Bool("fleet-desktop") {
+			done := make(chan struct{})
+			// TODO(lucas): Kill .app in the interrupt.
```

Comment:
Do we have to? Orbit should relaunch if interrupted, and the desktop app doesn't depend on orbit except to generate the auth token file.

If the file is not present, we just disable the menu item, or show a modal with the error, and they can be completely independent.

Reply:
Haven't thought this part through yet.

> Do we have to? Orbit should relaunch if interrupted, and the desktop app doesn't depend on orbit except to generate the auth token file.

We do if we want to allow auto-updating the "Fleet Desktop" application.

Reply:
Ah, right. That way we don't have to care about that logic in desktop too... makes sense!